kubernetes/llama-serve/README.md

```diff
@@ -0,0 +1,61 @@
+# vLLM serve
+
+This serves as a starting point to deploy the following models via vLLM:
+
+- `granite-8b`
+- `llama-3.1-70b`
+- `llama-3.2-3b`
+- `watt-8b`
+
+## Requirements
+
+A secret must be created to allow for the vLLM server to pull in a given model at startup time.
+
+```
+oc create secret generic huggingface-secret --from-literal=HF_TOKEN=hf_values
+```
+
+## Deploy
+
+You can find the deployment files for the different models in their respective directories:
+
+- [granite-8b](./granite-8b)
+    - [vllm.yaml](./granite-8b/vllm.yaml)
+- [llama3.1-70b](./llama3.1-70b)
+    - [vllm.yaml](./llama3.1-70b/vllm.yaml)
+- [llama3.2-3b](./llama3.2-3b)
+    - [vllm.yaml](./llama3.2-3b/vllm.yaml)
+- [watt-8b](./watt-8b)
+    - [vllm.yaml](./watt-8b/vllm.yaml)
+
+ Once you know which model you want to deploy, you can run the following:
+
+```
+oc create -f <your choice of model>/vllm.yaml
```

Comment:
I think this should be something `oc apply -f <your choice of model>`, that would get other artifacts like chat templates in config maps as well.